templates/conf.d/mime.types.epp

```diff
@@ -0,0 +1,6 @@
+# MANAGED BY PUPPET
+types {
+<% $nginx::mime_types.each |$key, $value| { -%>
+    <%= $key %> <%= $value %>;
+<% } -%>
+
```

Comment:
Please do not an empty newline.

Reply:
To solve a bit of the confusion I might have created. A quote from our [review docs](https://voxpupuli.org/docs/#reviewing-a-module-pr):
> * Files should always terminate with a newline if possible, with an exception being file or template fragments like those used with concat. This is the [POSIX 
standard](http://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap03.html#tag_03_206), and some tools don't handle the lack of a terminating newline properly